prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -27,10 +27,25 @@ asap_accepted_audiences = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_AU
 {{ end }}
 
 {{ if $ENABLE_XMPP_WEBSOCKET }}
+consider_bosh_secure = true;
+{{ end }}
+
+{{ $XMPP_CROSS_DOMAIN := $PUBLIC_URL }}
+{{ if eq (.Env.XMPP_CROSS_DOMAIN | default "false") "true"}}
+cross_domain_bosh = true
 -- Deprecated in 0.12
 -- https://github.com/bjc/prosody/commit/26542811eafd9c708a130272d7b7de77b92712de
-cross_domain_websocket = { "{{ $PUBLIC_URL }}" };
-consider_bosh_secure = true;
+{{ if $ENABLE_XMPP_WEBSOCKET }}
+cross_domain_websocket = true
+{{ end }}
```

Comment:
Same here! Do we need the if? Can we not just apply this setting even if websockets are disabled?

Reply:
yep, we sure can